uaclient/entitlements/landscape.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import os
+import subprocess
+from typing import Any, Dict, Optional, Tuple
+
+from uaclient import apt, event_logger, exceptions, messages, system, util
+from uaclient.entitlements.base import UAEntitlement
+from uaclient.entitlements.entitlement_status import ApplicationStatus
+
+event = event_logger.get_event_logger()
+
+LANDSCAPE_CLIENT_PACKAGE_NAME = "landscape-client"
+LANDSCAPE_CLIENT_CONFIG_PATH = "/etc/landscape/client.conf"
+LANDSCAPE_CLIENT_CONFIG_PATH_DISABLE_BACKUP = (
+    "/etc/landscape/client.conf.pro-disable-backup"
+)
+
+
+class LandscapeEntitlement(UAEntitlement):
+    name = "landscape"
+    title = "Landscape"
+    description = "Management and administration tool for Ubuntu"
+    help_doc_url = "https://ubuntu.com/landscape"
+
+    def _perform_enable(self, silent: bool = False) -> bool:
```

Comment:
Should we only print the messages if `silent` is False ? (I am recalling that I need to update some of PR regarding this as well)

Reply:
Actually, it seems we are completely ignoring this flag. We might discuss later what to do about it, but not on this PR